lib/internal/Magento/Framework/Lock/Backend/Database.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Framework\lock\Backend;
+
+class Database implements \Magento\Framework\Lock\LockManagerInterface
+{
+    /** @var \Magento\Framework\App\ResourceConnection */
+    private $resource;
+
+    public function __construct(
+        \Magento\Framework\App\ResourceConnection $resource
```

Comment:
Could you import this resource connection, it will be more readable there.

Reply:
Done.